test/langtools/tools/javap/8260403/T8260403.java

```diff
@@ -30,13 +30,21 @@
  * @modules jdk.jdeps/com.sun.tools.javap
  */
 import java.io.PrintWriter;
+import java.io.StringWriter;
 
 public class T8260403 {
 
     public static void main(String args[]) throws Exception {
-        if (com.sun.tools.javap.Main.run(new String[]{"-c", System.getProperty("test.classes") + "/InvalidSignature.class"},
-                new PrintWriter(System.out)) != 0) {
-            throw new AssertionError();
-        }
+        var sw = new StringWriter();
+        int res = com.sun.tools.javap.Main.run(
+                new String[]{"-c", System.getProperty("test.classes") + "/InvalidSignature.class"},
+                new PrintWriter(sw));
+        System.out.println(sw);
```

Comment:
is this for debug purpose only?

Reply:
It is the test fix.
8260403 says "javap should be more robust", however to expect zero exit code for invalid class is wrong.
It suppose to expect non-zero exit code and fail only on fatal errors or unhandled CP errors.